Atomic/syscontainers.py

```diff
@@ -183,6 +187,18 @@ def _check_oci_configuration_file(self, conf_path, remote=None):
         if configuration['root']['path'] != 'rootfs' and not remote:
             raise ValueError("Invalid configuration file.  Path must be 'rootfs'")
 
+        missing_source_paths = []
+        # Ensure that the source path specified in bind/rbind exists
+        if "mounts" in configuration:
+            for mount in configuration["mounts"]:
+                if not "type" in mount:
+                    continue
+                if "source" in mount and "bind" in mount["type"]:
```

Comment:
I wanted to prepare it in case we will have to consider other types of mount.  A missing type should always be skipped.  I can add a fixup path if you prefer the shorter version